snuba/admin/static/delete_tool/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Client from "SnubaAdmin/api_client";
+import React, { useEffect, useState } from "react"
+
+function DeleteTool(props: { api: Client }) {
+    const [storageName, setStorageName] = useState('')
+    const [columnConditions, setColumnConditions] = useState('')
+    const [result, setResult] = useState('')
+
+    return (
+        <div>
```

Comment:
also could you add a little description here about what the payload looks like ? That way someone could just copy paste it easily

Reply:
done